Source/2_Core/Replayer/ReplayerMenuLoader.cs

```diff
@@ -15,14 +16,39 @@
 namespace BeatLeader.Replayer {
     [PublicAPI]
     public class ReplayerMenuLoader : MonoBehaviour {
+        #region LoadData
+
+        internal class LoadData {
+            public enum Type {
```

Comment:
Instead of using Type enum, better implementation would be interface `ILoadData` with two classes `ScoreLoadData, FileLoadData`